webclient/package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "semistandard": "^11.0.0",
+    "vee-validate": "2.0.0-rc.18",
```

Comment:
I looked into this. Never heard of it before though I think it will work great. Nice add!